exporter/indexstats_collector_test.go

```diff
@@ -96,6 +97,67 @@ mongodb_testdb_testcol_02_idx_01_accesses_ops{key_name="idx_01",namespace="testd
 	assert.NoError(t, err)
 }
 
+func TestDescendingIndexOverride(t *testing.T) {
+	ctx, cancel := context.WithTimeout(context.Background(), 3*time.Second)
+	defer cancel()
+	client := tu.DefaultTestClient(ctx, t)
+	ti := labelsGetterMock{}
+	database := client.Database("testdb")
+	database.Drop(ctx)       //nolint:errcheck
+	defer database.Drop(ctx) //nolint:errcheck
+	for i := 0; i < 3; i++ {
+		collection := fmt.Sprintf("testcol_%02d", i)
+		for j := 0; j < 10; j++ {
+			_, err := database.Collection(collection).InsertOne(ctx, bson.M{"f1": j, "f2": "2"})
+			assert.NoError(t, err)
+		}
+		descendingMod := mongo.IndexModel{Keys: bson.M{"f1": -1}}
+		_, err := database.Collection(collection).Indexes().CreateOne(ctx, descendingMod)
+		assert.NoError(t, err)
+		ascendingMod := mongo.IndexModel{Keys: bson.M{"f1": 1}}
+		_, err = database.Collection(collection).Indexes().CreateOne(ctx, ascendingMod)
```

Comment:
both ascending and descending indexes are created on the same field in order to test the metrics and labels name.